### PR TITLE
DEVOXX-163: Add OpenSans font in devices that don’t support Roboto Medium

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java [hunk 1]
```diff
@@ -139,12 +140,26 @@ public void postInit(Scene scene) {
             }
         });
 
+        String deviceFactorSuffix = Services.get(DeviceService.class)
+                .map(s -> {
```
Comment (eugener) [inline DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: Much cleaner solution for search is a combination of filter and findFirst

Comment (eugener) [reply DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: But existing code should work

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: Would you say this is faster?
It requires a global String though. I don't see any advantage on it.

```
String deviceFactorSuffix = "";
        if (Platform.isAndroid()) {
            Services.get(DeviceService.class)
                .filter(s -> s.getModel() != null)
                .ifPresent(s -> {
                    for (String device : DevoxxSettings.DEVICES_WITH_SANS_CSS) {
                        if (s.getModel().toLowerCase(Locale.ROOT).startsWith(device)) {
                            deviceFactorSuffix = "_sans";
                        }
                    }
                });
        }
```

Comment (eugener) [reply DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: No.. I meant the device search.  
Something like.. ```DevoxxSettings.DEVICES_WITH_SANS_CSS.stream.filter.findFirst```, than you can do ```flatMap``` on the service itself. After that you get an ```Optional```, which can be converted into ```"_sans"``` or ```""```

All that will look cleaner IMO, but it should work fine as it is now

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: Once we have streams running, we'll apply that :)

Comment (eugener) [reply DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java]: Ah... not yet.. not yet 👍

**Pull request merged**